package.json

```diff
@@ -12,7 +12,8 @@
     "lint:ci": "eslint . --ext .ts",
     "lint:fix": "eslint . --ext .ts --fix",
     "build": "npm-run-all clean tsc copy-assets",
-    "pretest": "docker compose up -d db-test",
+    "pretest-docker": "docker compose up -d db-test",
+    "pretest-podman": "podman compose up -d db-test",
```

Comment:
Can we revert this as well please? Should be same solution (alias docker=podman)